src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,82 @@
+.watch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-left: 0;
+  color: black;
+
+  @include circle($size-watch);
+  box-sizing: border-box;
+  border: 5px solid black;
```

Comment:
it's good practice move color to variables. Please improve your code style according this comment not only in these line but on whole your solution